src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -49,6 +47,42 @@
 
 redshift_dialect.replace(WellKnownTextGeometrySegment=Nothing())
 
+ObjectReferenceSegment = redshift_dialect.get_segment("ObjectReferenceSegment")
```

Comment:
I don’t think you even need this. Other dialects use this without importing it.

```suggestion
```

Reply:
@tunetheweb you do: if you don't do something like this, you get an `not defined` error:
```
Traceback (most recent call last):
  File "test/generate_parse_fixture_yml.py", line 51, in <module>
    main()
  File "test/generate_parse_fixture_yml.py", line 46, in main
    for _ in pool.imap_unordered(generate_parse_fixture, parse_success_examples):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/multiprocessing/pool.py", line 865, in next
    raise value
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/test/generate_parse_fixture_yml.py", line 13, in generate_parse_fixture
    tree = parse_example_file(dialect, sqlfile)
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/test/conftest.py", line 100, in parse_example_file
    config = FluffConfig(overrides=dict(dialect=dialect))
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/.tox/generate-fixture-yml/lib/python3.8/site-packages/sqlfluff/core/config.py", line 506, in __init__
    self._configs["core"]["dialect_obj"] = dialect_selector(
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/.tox/generate-fixture-yml/lib/python3.8/site-packages/sqlfluff/core/dialects/__init__.py", line 77, in dialect_selector
    dialect = load_raw_dialect(s or "ansi")
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/.tox/generate-fixture-yml/lib/python3.8/site-packages/sqlfluff/core/dialects/__init__.py", line 53, in load_raw_dialect
    return getattr(import_module(f"{base_module}.{module}"), name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/michaelrosenberg/personal/repos/sqlfluff/.tox/generate-fixture-yml/lib/python3.8/site-packages/sqlfluff/dialects/dialect_redshift.py", line 50, in <module>
    class ColumnReferenceSegment(ObjectReferenceSegment):
NameError: name 'ObjectReferenceSegment' is not defined
ERROR: InvocationError for command /Users/michaelrosenberg/personal/repos/sqlfluff/.tox/generate-fixture-yml/bin/python test/generate_parse_fixture_yml.py (exited with code 1)
```

You have to find some way to make the object available in the namespace of the `dialect_redshift.py` module.

Reply:
Hmmm… I wonder how come this works in BigQuert version without doing this?

Reply:
the BigQuery Dialect does do this: https://github.com/PLBMR/sqlfluff/blob/plbmr-issue_1672_support_super_data_types/src/sqlfluff/dialects/dialect_bigquery.py#L677

Should I follow this pattern?

Reply:
Oh missed that. Yes I guess so.

Reply:
Sounds good! Will follow this pattern.

Reply:
Done!